### DISCUSSION TITLE
Question regarding chat message

### DISCUSSION THREAD

**Discussion by skymemoryGit** [fraxiinus/ReplayBook#176] [Feature Request]
Hi, I am working on an NLP project and I find this repository very interesting and I would like to ask you a  question. 
Is possible to extract the sequence message in the chat section  (eg: pings message, all chat,, etc..)? 
Would be really interesting if this information could be extracted and reported in json file

**Comment by fraxiinus** [comment]
Hi, unfortunately I believe pings and messages are part of the 'payload' of a replay, while the part we are able to read is the 'header' 
I don't think there is even a count of total pings or messages in the stats data.
